microsoft-r/install/machine-learning-server-windows-packages.md

```diff
@@ -108,4 +108,4 @@ conda uninstall <packagename>
 + [What's new in Machine Learning Server](../whats-new-in-machine-learning-server.md)
 + [Supported platforms](r-server-install-supported-platforms.md)  
 + [Known Issues](../resources-known-issues.md)  
-+ [Configure Machine Learning Server to operationalize your analytics](../what-is-operationalization.md)
\ No newline at end of file
++ [Configure Machine Learning Server to operationalize your analytics](../what-is-operationalization.md)
```

Comment:
what have you changed here?

Reply:
The only thing I can think of was I accidentally added a newline to the end of the file.  The master branch had 111 lines and my commit had 112.  I've pushed a new commit that removes it.